File: work/nh/projects/nh_it_apps/client/src/app/models/Storage.ts
export class Storage {
    id: number = 0;
    serverId : number = 0;
    type: string = "";
    path: string = "";
    totalCapacityGB: number = 0;
    comments: string = "";
    lastUpdatedBy: string = "";
    lastUpdated: Date = new Date();
}

File: 🏢work/nh/projects/dev/nh_it_apps/api/Models/Entities/Storage.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models.Entities
{
    public class Storage : BaseEntity
    {
        public int ServerId { get; set; }

        [Column(TypeName="varchar(20)")]
        public string Type { get; set; }

        [Column(TypeName="varchar(250)")]
        public string Path { get; set; }
        public int TotalCapacityGB { get; set; }
    }
}
